packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/index.js

```diff
@@ -1,17 +1,66 @@
-import React from 'react';
-import { SettingsPageTitle } from '@strapi/helper-plugin';
-import { HeaderLayout } from '@strapi/design-system/Layout';
+import React, { useState } from 'react';
+import { SettingsPageTitle, DynamicTable } from '@strapi/helper-plugin';
+import { HeaderLayout, ContentLayout } from '@strapi/design-system/Layout';
 import { Main } from '@strapi/design-system/Main';
 import { useIntl } from 'react-intl';
+import TableRows from './DynamicTable/TableRows';
+import tableHeaders from './utils/tableHeaders';
+import ModalDialog from './ModalDialog';
+
+const data = [
+  {
+    id: 1,
+    action: 'Update',
+    date: '2022-11-14T23:04:00.000Z',
+    user: 'John Doe',
+  },
+  {
+    id: 2,
+    action: 'Create',
+    date: '2022-11-04T18:24:00.000Z',
+    user: 'Kai Doe',
+  },
+  {
+    id: 3,
+    action: 'Delete',
+    date: '2022-10-09T11:26:00.000Z',
+    user: 'John Doe',
+  },
+  {
+    id: 4,
+    action: 'Log in',
+    date: '2022-10-09T11:24:00.000Z',
+    user: 'Kai Doe',
+  },
+];
 
 const ListView = () => {
   const { formatMessage } = useIntl();
+  const [openModal, setOpenModal] = useState(false);
+  const [detailsActionData, setActionData] = useState(null);
```

Comment:
Could you make the setter function match the match of the 1st variable? so `setDetailsActionData`